src/test/datascience/editor-integration/codewatcher.unit.test.ts

```diff
@@ -550,10 +553,14 @@ testing2`;
                     TypeMoq.It.isAny()
                 )
             )
-            .returns(() => Promise.resolve(true))
+            .returns(() => {
+                deferred.resolve();
+                return Promise.resolve(true);
+            })
             .verifiable(TypeMoq.Times.once());
 
         await codeWatcher.runAllCells();
+        await deferred.promise;
```

Comment:
The unit test did change here. The command now returns when the code has all been queued, which I believe makes sense for the command.